app/services/admin/data_counts.rb

```diff
@@ -0,0 +1,41 @@
+module Admin
+  class DataCounts
+    Response = Struct.new(
+      :open_abuse_reports_count,
+      :possible_spam_users_count,
+      :flags_count,
+      :flags_posts_count,
+      :flags_comments_count,
+      :flags_users_count,
+      keyword_init: true,
+    )
+
+    # @return [Admin::DataCounts::Response]
+    def self.call
+      # If this class is to be re-used in more situations, we could pass in params
+      # to dictate which counts are needed, to reduce unneeded queries.
+
+      open_abuse_reports_count =
+        FeedbackMessage.open_abuse_reports.size
+
+      possible_spam_users_count = User.registered.where("length(name) > ?", 30)
+        .where("created_at > ?", 48.hours.ago)
+        .order(created_at: :desc)
+        .select(:username, :name, :id)
+        .where.not("username LIKE ?", "%spam_%")
+        .size
+
+      flags = Reaction
+        .includes(:user, :reactable)
+        .privileged_category
+      Response.new(
+        open_abuse_reports_count: open_abuse_reports_count,
+        possible_spam_users_count: possible_spam_users_count,
+        flags_count: flags.size,
+        flags_posts_count: flags.where(reactable_type: "Article").size,
+        flags_comments_count: flags.where(reactable_type: "Article").size,
+        flags_users_count: flags.where(reactable_type: "User").size,
```

Comment:
```
flags_posts_count: flags.where(reactable_type: "Article").size,
flags_comments_count: flags.where(reactable_type: "Article").size,
```

The above two properties, with different names, will always have the same value.  Is that what we want?

Reply:
I _think_ that's actually just a typo, and is, in fact, a bug if it ships.

Should read `flags.where(reactable_type: "Comment").size`, unless I'm confused.

Reply:
Ok, I'm _blindly_ fixing it, I hope it works! :)

Reply:
LGTM

And now for the Travis issues.